navigation/stack_navigators/AuthStack.js

```diff
@@ -16,6 +18,8 @@ export default function AuthStackNavigator() {
       }}>
       <AuthStack.Screen name="Onboarding" component={OnboardingScreen} />
       <AuthStack.Screen name="PhoneNumber" component={PhoneNumberScreen} />
+      <AuthStack.Screen name="Permissions" component={PermissionsScreen} />
+      <AuthStack.Screen name="StoreSelect" component={StoreSelectScreen} />
```

Comment:
Not sure it would make more sense to include StoreSelect in StoresStack instead since this screen should still be usable after the initial onboarding process (for modifying selections later on)